### ISSUE TITLE
Allow requiring collapsed blocks for directive comments

### ISSUE THREAD

**Issue by haykam821** [mysticatea/eslint-plugin-eslint-comments#67]
I would like to enforce directive comments to use block comment syntax on a single line, such as the following:

```js
/* eslint-disable no-undef */
```

The following styles should be disallowed:

```js
// eslint-disable no-undef
/*
  eslint-disable no-undef
*/
```

Comments that are not directive comments should not be affected.

**Comment by MichaelDeBoey** [comment]
Hi @haykam821!

Since this repo is unmaintained, you might want to re-open this issue in the @eslint-community fork https://github.com/eslint-community/eslint-plugin-eslint-comments

For more info about why we created this organization, you can read https://eslint.org/blog/2023/03/announcing-eslint-community-org
